requirements.txt

```diff
@@ -9,12 +10,22 @@ scikit-bio
 samtools
 bedtools
 bowtie2
+biopython
+diamond
+hdbscan
 hmmer
+nextflow>=20.10
+numpy
+pandas>=1.1
 prodigal
 make
 nextflow
 parallel
 requests
+rsync
+scikit-learn
+scikit-bio
+samtools
+tqdm
+tsne
 umap-learn
-hdbscan
-attrs
```

Comment:
The following dependencies are mentioned twice:
`biopython`, `pandas`, `tsne`, `tqdm`, `skit-learn`, `skit-bio`, `samtools`, `nextflow`, `numpy`,